src/_http/index.js

```diff
@@ -169,6 +169,8 @@ function validateTags(tags) {
   validateTagValues(Object.values(tags))
 }
 
+const AlphaNumPattern = new RegExp(/^\w+$/)
```

Comment:
try putting this at line 6 and i think it should still work mint it just once for any call to HttpClient.

Reply:
Yeah, I wasn't sure - felt weird there as a convention with all the imports. Should I make it all caps?

Reply:
yeah that sounds like a good call.

Reply:
you got another regex at line 147 btw.

Reply:
actually reading a bit more: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

it sounds like a literal will cause it to be compiled once.

So we may be good.